docs/configuration/acme.md

```diff
@@ -140,8 +140,7 @@ entryPoint = "https"
     These are Let's Encrypt limitations as described on the [community forum](https://community.letsencrypt.org/t/support-for-ports-other-than-80-and-443/3419/72).
 
 !!! note
-    Wildcard certificates can be generated only if `acme.dnsChallenge`
-option is enable.
+    Wildcard certificates can be generated only if `acme.dnsChallenge` option is enable.
```

Comment:
Wildcard certificates can only be generated if the `acme.dnsChallenge` option is enabled.